### PR TITLE
Adding links to the new workshops on landing pages

### PR DESCRIPTION
**Pull request by jrastaban**
Add secondary CTAs on landing pages that point to the new Live Workshops. This is a lift and shift from the workshop pages so a pass through to update the language would be nice.

**Note:** Please fill out the PR Template to ensure proper processing.

Closes: #

## Summary

Add secondary CTAs on landing pages that point to the new Live Workshop signups. 

## Effect

The following changes are already committed:

- CTA's are added to 4 new landing pages

## Remaining Work

The following changes still need to be completed:

- [ ] A pass for language

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/data/landings.yaml [hunk 1]
```diff
@@ -687,6 +699,10 @@
     title: 'Front End Performance Guide'
     subtitle: "Ready to dive into Pantheon's Global CDN? Check out this guide to learn how to ace an online speed test."
     url: '/guides/frontend-performance'
+  cta_alt:
+    title: 'Website Performance with Varnish, Redis, and New Relic'
+    subtitle: 'How does Pantheon help your website be faster than other platforms?'
```
Comment (sparklingrobots) [inline source/data/landings.yaml:704]: Minor thing: Maybe 'How does Pantheon help your website run faster than other platforms?'

**Comment by davidneedham** [comment]
Hmm, I wonder if this is going to generate a page for us to review or if @alexfornuto has to work some magic first.

File: source/data/landings.yaml (hunk 1 above)
Comment (jrastaban) [reply source/data/landings.yaml]: I like it, added. (but forgot to mention it in the commit)

**Comment by jrastaban** [comment]
It isn't obvious for these that you are going to a workshop/demo. Would it be worth adding it to the titles? Some of them like "Website Performance with Varnish, Redis, and New Relic" are already so long I didn't include initially it but I feel like the context is needed.

**Comment by alexfornuto** [comment]
@davidneedham: Preview wasn't generated because this came from a fork and not the main repo. See #5613 for preview build.

@jrastaban: I agree that we want to specify Live Workshop, and did so in my edits. In the example you noted, I truncated the title to compensae.

**Comment by alexfornuto** [comment]
Merged as part of #5613.

**Pull request closed without merging**